velox/core/PlanNode.h

```diff
@@ -1936,4 +1936,53 @@ class WindowNode : public PlanNode {
   const RowTypePtr outputType_;
 };
 
+/// Optimized version of a WindowNode for a single row_number function with an
+/// optional limit and no sorting.
```

Comment:
It seems like even partition keys could be empty. Please can you add this info to the comment.

Reply:
@aditi-pandit Sure. Updated.